docs/CNVs/CNV-methods.tex

```diff
@@ -482,6 +497,134 @@ \subsection{Using Panel of Normals for Allelic Fraction Model} \label{allelic-Po
 \end{equation}
 %
 where $f$ and $\pi$ may once again be sampled via adaptive Metropolis.
+
+\subsection{Generative Model for Read Counts} \label{coverage-model}
+In this section we address several goals:
+
+\begin{enumerate}
+
+\item Connect copy ratio (or copy number) and raw read counts in a single probabilistic model without transforming the data.
+
+\item Take into account the Poisson nature of coverage depth, thereby giving less weight to low-coverage targets and separating the inherent variance due to Poisson statistics from experimental noise.  We want to use the panel of normals to subtract only the latter.
+
+\item Choose the number of principal components to use in an automatic and principled manner.
+
+\item Use an algorithm that does not waste time calculating all principal components when we only want the few most significant ones.
+
+\item Correct for CNV events that occur in the panel of normals.
+
+\end{enumerate}
+
+Suppose we have vectors of read counts over a set of $T$ targets for $S$ samples, $\vn_s, \, s = 1 \ldots S$ where $n_{st}$ is the coverage of sample $s$ at target $t$.  We imagine that laboratory conditions for a particular sample yielding an underlying bias vector $\vb_s$, where $e^{b_{st}}$ is the propensity of target $t$ to be captured, sequenced, and mapped in the preparation of sample $s$.  Suppose also that sample $s$ has an average depth $d_s$ and a vector of copy numbers $\vc_s$, where the latent variable $c_{st}$ is the copy number of sample $s$ at target $t$.  Our model for coverage is:
+%
+\begin{equation}
+n_{st} \sim {\rm Poisson}(d_s c_{st} e^{b_{st}})
+\end{equation}
+%
+We can achieve many of the goals listed above by performing probabilistic PCA not on $\vn$ directly but rather on $\vb$.  This is one reason why we choose a Poisson parameter proportional to $e^b$ and not simply $b$ -- probabilistic PCA involves a normal distribution on $\vb_s$, whereas the Poisson parameter must be positive.  Furthermore, since PCA is a linear model with additive effects of different latent features, using $e^b$ represents multiplicative effects, which is more plausible.  Our model is
```

Comment:
Naive question: what is the fundamental advantage of doing PCA over let's say, a naive MLE of the multiplicative bias and copy number at each target? we still get a super simple generative model (Eq. 42) without any latent variables. By doing PCA, we are tacitly assuming that the biases at different targets are strongly correlated and can be described via an affine mapping from a lower dimensional manifold. Is this a good assumption?  Are we doing PCA because we do not have enough data to infer the multiplicative bias at each target separately, or do we gain something more profound from PCA? educate me please, I'm sure these questions stem from my lack of insight.